crates/matrix-sdk-base/src/rooms/mod.rs

```diff
@@ -116,6 +128,34 @@ pub struct BaseRoomInfo {
     pub(crate) notable_tags: RoomNotableTags,
 }
 
+#[derive(Clone, Debug, Default, Serialize, Deserialize)]
+pub struct RawRoomInfo {
```

Comment:
also, question: does it need to be public so that downstream implementations of the state store trait can reuse it? (If so, please indicate so in the comment.)

I wonder if it still needs that, with my suggestion above to use `serde(from / into)`.